### ISSUE TITLE
Install instruction doesn't work, return gpg: keyserver receive failed: Network is unreachable

### ISSUE THREAD

**Issue by adonis-wyc** [rvm/rvm#5291]
gpg export commands fail even with the 3 alternative URLs.
<img width="765" alt="image" src="https://user-images.githubusercontent.com/41361698/212322551-86e221e8-ba2a-4d1c-b0e0-b7e84508a49a.png">
The response is here:
<img width="1481" alt="image" src="https://user-images.githubusercontent.com/41361698/212323053-ce3cc37b-f0f7-409e-a240-6aeb61f170e1.png">

**Comment by mathieujobin** [comment]
the workaround I found is to use a different server

`gpg2 --keyserver keyserver.ubuntu.com --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB`

**Comment by mathieujobin** [comment]
maybe we should update the website... ? @pkuczynski ?

**Comment by thomas-outing** [comment]
I agree. The website should be updated. I ran into this issue as well and the solution was to use the server specified above.

**Comment by pkuczynski** [comment]
> maybe we should update the website... ? @pkuczynski ?

Absolutely we should. Deployment fails currently. Would you find some time to work on https://github.com/rvm/rvm-site/pull/346 @mathieujobin ?

**Comment by mathieujobin** [comment]
FYI - It appears this is caused by gpg default server being down for like ~2 years

See last comment on https://unix.stackexchange.com/a/530784/29312
and git commit

https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=blobdiff;f=configure.ac;h=e651db8a0df4f8ad76f08a0df64d1e099ca77938;hp=0bdd90ed728aa56bbd31719141bfe555b90950fb;hb=47c4e3e00;hpb=5fe4b978875271fb55f1f674ab545bed2b97a7a8

upgrading to latest gpg should work
and make sure you don't have it hardcoded in your config

```bash
$ grep keyserver ~/.gnupg/gpg.conf | sed 's/#.*$//'  | sort -u 

$
```

just in case https://github.com/rvm/rvm-site/pull/365

**Comment by RyeGordo** [comment]
Has the server changed again?
```
gpg --keyserver keyserver.ubuntu.com --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
```
does not to work.

> gpg: keyserver receive failed: Network is unreachable

**Comment by JaneCoder** [comment]
None of the solutions here work for me. I have the latest gpg2 and have tried all the possible servers I can find here.
